packages/web-components/src/stories/astro-uxds/welcome/vue.stories.mdx

```diff
@@ -123,6 +123,21 @@ module.exports = {
   </rux-tab-panel>
 </rux-tab-panels>
 
+### Disable eslint rule
+Be aware that if you try to pass in any native HTML elements as slots using the `slot` attribute. 
```

Comment:
Maybe I'm losing my mind but I don't think this sentence makes sense?

Reply:
bah thank you! fixed!